website/blog/2020-03-26-version-0.62.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Announcing React Native 0.62 with Flipper
+author: Rick Hanlon
+authorTitle: React Native Core at Facebook
+authorURL: https://twitter.com/rickhanlonii
+authorImageURL: https://avatars3.githubusercontent.com/u/2440089?s=460&v=4
+authorTwitter: rickhanlonii
+category: announcements
+---
+
+Today we’re releasing React Native version 0.62 which includes support for Flipper by default.
+
+This release comes in the midst of a global pandemic. We’re releasing this version today to respect the work of hundreds of contributors who made this release possible and to prevent the release from falling too far behind master. Please be mindful of the reduced capacity of contributors to help with issues and prepare to delay upgrading if necessary.
+
+## Flipper by default
+
+[Flipper](https://fbflipper.com/) is a developer tool for debugging mobile apps. It’s popular in the Android and iOS communities, and in this release we’ve enabled support by default for new and existing React Native apps.
+
+![Screenshot of Flipper for React Native](/blog/assets/0.62-flipper.png)
+
+Flipper provides the following features out of the box:
+
+- **Metro Actions**: Reload the app and triggering the Dev Menu right from the toolbar.
+- **Crash Reporter**: View the crash reports from Android and iOS devices for all crashes.
```

Comment:
```suggestion
- **Crash Reporter**: View crash reports from Android and iOS devices.
```

Unsure if the “all crashes” had any significance?

Reply:
Honestly was just fluffing the content so the line ended close to all the others

Reply:
Ahaha, I can respect that 😄 

![](https://media.giphy.com/media/a9wFQSc0oRQGI/giphy.gif)